Meetings/Meeting.php

```diff
@@ -0,0 +1,282 @@
+     *
+     * @var int
+     */
+    private $milestoneId;
+
+    /**
+     * Meeting custom fields
+     *
+     * @var array
+     */
+    private $customFields;
+
+    /**
+     * @var array
+     */
+    private $attendees;
+
+    /**
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param int $id
+     */
+    public function setId($id)
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * @return int
+     */
+    public function getStartDate()
+    {
+        return $this->startDate;
+    }
+
+    /**
+     * @param int|\DateTime $startDate
+     */
+    public function setStartDate($startDate)
+    {
+        if ($startDate instanceof \DateTime) {
+            $startDate = $startDate->getTimestamp();
+        }
+        $this->startDate = $startDate;
+    }
+
+    /**
+     * @return int
+     */
+    public function getUserId()
+    {
+        return $this->userId;
+    }
+
+    /**
+     * @param int $userId
+     */
+    public function setUserId($userId)
+    {
+        $this->userId = $userId;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    /**
+     * @param string $title
+     */
+    public function setTitle($title)
+    {
+        $this->title = $title;
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getDescription()
+    {
+        return $this->description;
+    }
+
+    /**
+     * @param null|string $description
+     */
+    public function setDescription($description)
+    {
+        $this->description = $description;
+    }
+
+    /**
+     * @return int
+     */
+    public function getDuration()
+    {
+        return $this->duration;
+    }
+
+    /**
+     * @param int $duration
+     */
+    public function setDuration($duration)
+    {
+        $this->duration = $duration;
+    }
+
+    /**
+     * @return int
+     */
+    public function getMilestoneId()
+    {
+        return $this->milestoneId;
+    }
+
+    /**
+     * @param int $milestoneId
+     */
+    public function setMilestoneId($milestoneId)
+    {
+        $this->milestoneId = $milestoneId;
+    }
+
+    /**
+     * Set a single custom field
+     *
+     * @param string $id
+     * @param mixed  $value
+     */
+    public function setCustomField($id, $value)
+    {
+        $this->customFields[$id] = $value;
+    }
+
+    /**
+     * @return array
+     */
+    public function getCustomFields()
+    {
+        return $this->customFields;
+    }
+
+    /**
+     * @param array $customFields
+     */
+    public function setCustomFields($customFields)
+    {
+        $this->customFields = $customFields;
+    }
+
+    /**
+     * @param $data
+     * @return Meeting
+     */
+    public static function initializeWithRawData($data)
+    {
+        $meeting = new Meeting();
+        foreach ($data as $key => $value) {
+            switch ($key) {
```

Comment:
you only have a default, why do you use a switch case?

Reply:
Copied it from Product.php tested it and it worked. But shouldn't be there 👍 

Removed in 5a1e46d9a38e2f22b76e11d07bf86eaaff39efe9